packages/LUIS/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "luis-apis",
-  "version": "3.0.0",
+  "version": "2.6.2",
```

Comment:
Did we pin in the package.json? Or do we need this change? When I don't change package dependencies I try to avoid any package-lock.json changes...